fg-stitch-lib/src/align/aligners/mod.rs

```diff
@@ -30,6 +30,11 @@ use crate::{
     util::{
         dna::reverse_complement,
         index_map::IndexMap,
+        tag::{
```

Comment:
Fine for now, but looking at it again, this is probably a place where you want to introduce eg `enum CustomTags` and implement `From<CustomTag> for Tag`.